Decisions.Equifax/EquifaxSettings.cs

```diff
@@ -18,25 +18,33 @@ namespace Decisions.Equifax
     public class EquifaxSettings : AbstractModuleSettings, IInitializable, IValidationSource
     {
         [ORMField]
-        [PropertyClassification(0, "OAuth Url", "OAuth Settings")]
+        [PropertyClassification(new string[] {"OAuth Settings"} , "OAuth URL",4)]
         public string EquifaxOAuthUrl { get; set; }
 
         [ORMField]
-        [PropertyClassification(1, "API Client ID", "OAuth Settings")]
+        [PropertyClassification(new string[] {"OAuth Settings"} , "API Client ID", 5)]
         public string EquifaxClientId { get; set; }
 
         [ORMField]
-        [PropertyClassification(2, "API Client Secret", "OAuth Settings")]
+        [PropertyClassification(new string[] {"OAuth Settings"} , "API Client Secret", 6)]
         public string EquifaxClientSecret { get; set; }
-        
```

Comment:
Missing a blank line here. Lets take a final pass over all of this to 
1. Remove unused using statements (maybe not this file, but seen a few that I may have left also)
2. Make sure our spacing between methods and properties is correct. Since this module is shared in public, lets make it pretty.

Reply:
Gotcha